src/Acuminator/Acuminator.Analyzers/Analyzers/BQL/BqlParameterMismatch/BqlParameterMismatchAnalyzer.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Acuminator.Utilities;
+using PX.Data;
+
+
+namespace Acuminator.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public partial class BqlParameterMismatchAnalyzer : PXDiagnosticAnalyzer
+    {
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics =>
+            ImmutableArray.Create(Descriptors.PX1015_PXBqlParametersMismatch);
+
+        internal override void AnalyzeCompilation(CompilationStartAnalysisContext compilationStartContext, PXContext pxContext)
+        {
+            compilationStartContext.RegisterSyntaxNodeAction(c => AnalyzeNode(c, pxContext), SyntaxKind.InvocationExpression);
+        }
+
+        private static void AnalyzeNode(SyntaxNodeAnalysisContext syntaxContext, PXContext pxContext)
+        {
+            if (syntaxContext.CancellationToken.IsCancellationRequested || !(syntaxContext.Node is InvocationExpressionSyntax invocationNode))
+                return;
+
+            var symbolInfo = syntaxContext.SemanticModel.GetSymbolInfo(invocationNode);
+
+            if (!(symbolInfo.Symbol is IMethodSymbol methodSymbol) || !IsValidMethodGeneralCheck(methodSymbol, pxContext))
+                return;
+
+            if (methodSymbol.IsStatic)
+                AnalyzeStaticInvocation(methodSymbol, pxContext, syntaxContext, invocationNode);
+			else
```

Comment:
Formatting is broken completely

Reply:
I'll fix. Probably forgot to switch tabs/spaces at my home VS